x-pack/plugins/endpoint/public/applications/endpoint/view/alerts/alert_details_overview.tsx

```diff
@@ -0,0 +1,338 @@
+        title: 'MD5',
+        description: 'TODO',
+      },
+      {
+        title: 'SHA1',
+        description: 'TODO',
+      },
+      {
+        title: 'SHA256',
+        description: 'TODO',
+      },
+    ];
+  }, []);
+
+  const fileDetailsColumns = useMemo(() => {
+    return [
+      {
+        title: 'File Name',
+        description: 'TODO',
+      },
+      {
+        title: 'File Path',
+        description: 'TODO',
+      },
+      {
+        title: 'File Size',
+        description: 'TODO',
+      },
+      {
+        title: 'File Created',
+        description: 'TODO',
+      },
+      {
+        title: 'File Modified',
+        description: 'TODO',
+      },
+      {
+        title: 'File Accessed',
+        description: 'TODO',
+      },
+      {
+        title: 'Signer',
+        description: 'TODO',
+      },
+      {
+        title: 'Owner',
+        description: 'TODO',
+      },
+    ];
+  }, []);
+
+  const sourceProcessDetailsColumns = useMemo(() => {
+    return [
+      {
+        title: 'Process ID',
+        description: 'TODO',
+      },
+      {
+        title: 'Process Name',
+        description: 'TODO',
+      },
+      {
+        title: 'Process Path',
+        description: 'TODO',
+      },
+      {
+        title: 'MD5',
+        description: 'TODO',
+      },
+      {
+        title: 'SHA1',
+        description: 'TODO',
+      },
+      {
+        title: 'SHA256',
+        description: 'TODO',
+      },
+      {
+        title: 'MalwareScore',
+        description: 'TODO',
+      },
+      {
+        title: 'Parent Process ID',
+        description: 'TODO',
+      },
+      {
+        title: 'Signer',
+        description: 'TODO',
+      },
+      {
+        title: 'Username',
+        description: 'TODO',
+      },
+      {
+        title: 'Domain',
+        description: 'TODO',
+      },
+    ];
+  }, []);
+
+  const sourceProcessTokenDetailsColumns = useMemo(() => {
+    return [
+      {
+        title: 'SID',
+        description: 'TODO',
+      },
+      {
+        title: 'Integrity Level',
+        description: 'TODO',
+      },
+      {
+        title: 'Privileges',
+        description: 'TODO',
+      },
+    ];
+  }, []);
+
+  return (
+    <>
+      {/* Hard coded top level alert details component. TODO: maybe abstract somewhere else? */}
+      <EuiTitle size="s">
+        <h3>Detected Malicious File</h3>
+      </EuiTitle>
+      <EuiSpacer />
+      <EuiText>
+        <p>
+          Endgame MalwareScore detected the opening of a document with a blah blah blah on{' '}
+          {alertDetailsData.host.hostname} on {AlertDetailsDate}
+        </p>
+      </EuiText>
+      <EuiSpacer />
+      <EuiText>
+        Endpoint Status: <EuiHealth color="success">Online</EuiHealth>
+      </EuiText>
+      <EuiSpacer />
+      <EuiText>Alert Status: Open</EuiText>
+      <EuiSpacer />
+
+      {/* Start of Alert Details overview component TODO: delete this comment eventually */}
+      <EuiAccordion
```

Comment:
I think we should pull these out into their own components because we'll have to reuse them in the near future when we add more alert types. The new components could take `isOpen` as a prop and pass that down to `initialIsOpen`.